src/lookup-table/services/LookupTable.service.ts

```diff
@@ -183,7 +189,7 @@ const LookupTableService = async (props: LookupTableServiceProps) => {
       (obj: any) => dbFieldMap.filter((otherObj: any) => otherObj.label == obj.label).length === 0
     )
 
-    await lookupTableModel.updateStructureFieldMaps(tableName, fieldMaps)
+    await lookupTableModel.updateStructureFieldMaps(tableName, name, fieldMaps, dataViewCode)
```

Comment:
Yes, I've updated the front-end so that the code and the "display" table name can be updated when re-importing a lookup table. (The underlying true table name never changes though)